### DISCUSSION TITLE
Should I update the firmware? (Miyoo Plus)

### DISCUSSION THREAD

**Discussion by dreams897** [OnionUI/Onion#1062] [General]
I have a Miyoo Plus with firmware version 202303262339. Should I update to a newer one on lomiyoo.com? I'm seeing 0505 firmware and 0419 firmware listed. Is there any benefit in updating? Will it even be compatible with Onion?

**Comment by Aemiii91** [comment]
We have a section in the wiki about firmware here: https://github.com/OnionUI/Onion/wiki/Installation#check-your-firmware

You can stay on `0326` if you want, but `0505` and `0609` contain a minor fix that'll reduce the risk of SD card corruption. These versions are for Mini Plus only.

`20220419` is for the regular Mini only.

**Reply by Phanturian** [reply]
Is there a safe place to download 0505 and 0609? The lomiyoo website triggers high risk warnings on every browser and device I try. I've been trying to determine if the risk is worth it. If anyone can shed light on whether the lomiyoo website is actually safe, or if there is a safer repository for these firmware upgrades I would greatly appreciate it.

**Reply by dreams897** [reply]
Honestly I just used Tor via Brave Browser. I don't think these firmware files are available anywhere else. I also could not find 0609 so not even sure what that is

**Reply by Aemiii91** [reply]
They are also available here: https://github.com/anzz1/miyoomini-resources/tree/master/firmware/354

**Reply by Phanturian** [reply]
Thank you very much, this is what I was hoping for.
